### DISCUSSION TITLE
請教VcPrimitiveModel載入問題

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#349] [Q&A] [answered]
您好,
VcPrimitiveModel載入是否有分CDN與組件方式的不同?
原先開發時單獨使用組件方式,載入GLB檔是可以正常顯示MODEL
但用相同的程式碼與設置加到項目後,並使用CDN做開發時,
MODEL無法顯示,放大了幾萬倍後才發現座標應該有問題
整個模型被包在地球裡了
請問這要調整那裡?
(註:項目裡用Cesium方法viewer.scene.primitives.add(Cesium.Model.fromGltf(
是可以正確載入3D模型)
程式碼
`        <vc-primitive-model ref="primitive"
                            :url="url"
                            :modelMatrix="modelMatrix"
                            :scale="100010"
                            @@click="onClicked">
        </vc-primitive-model>`

![2023-02-14_145855](https://user-images.githubusercontent.com/5283322/218663036-35e10ca9-1177-4cc6-bf86-38f71fa75135.png)

**Comment by zouyaoji** [comment]
没区别哈。
我怀疑是 modelMatrix 没给对。

**Comment by stan88** [comment, marked as answer]
這個問題改小寫也解決了

**Reply by zouyaoji** [reply]
好的
